plugins/network/multibandwidth

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+: <<=cut
+
+=head1 NAME
+
+multibandwidth - Plugin to monitor the bandwidth between localhost and serveral hosts.
+
+=head1 APPLICABLE SYSTEMS
+
+This plugin is useful when you want to monitor the bandwidth between localhost and multiple hosts in a single graph or in several graphs making different soft links.
+
+=head1 REQUIREMENTS
+
+bing installed.
+
+You can install bing by using (Ubuntu/Debian): apt-get install bing
+
+=head1 CONFIGURATION
+
+The following is the default configuration
+
+[multibandwidth]
+user root
+env.hosts example.org example2.org example3.org
+env.samples 10
+env.small_packet_size 1000
+env.big_packet_size 8000
+
+- env.hosts explanation: hostname or IP of the hosts to calculate the bandwidth.
+
+- env.samples explanation: Reset stats after sending samples ECHO_REQUEST packets.
+
+- env.small_packet_size explanation: Specifies the number of data bytes to be sent in the small packets. The default and minimum value is 44.
+
+- env.big_packet_size explanation: Specifies the number of data bytes to be sent in the big packets. The default is 108. The size should be chosen so that big packet roundtrip times are long enough to be accurately measured.
+
+
+=head1 MAGIC MARKERS
+
+#%# capabilities=autoconf
+
+=head1 VERSION
+
+1.1.17
+
+=head1 AUTHOR
+
+Jose Manuel Febrer Cortés <https://www.linkedin.com/in/jfebrer/>
+Marco Bertola's help <https://www.linkedin.com/in/bertolamarco/>
+
+=head1 LICENSE
+
+GPLv2
+
+=cut
+
+
+
+case $1 in
+   config)
+                echo graph_title MultiBandwidth
+                echo 'graph_vlabel Mbps'
+                echo 'graph_args --base 1000 -l 0'
+                echo 'graph_scale no'
```

Comment:
See below for my comment regarding `Mbps`.
You should switch to the base unit (bytes) and allow automatic scaling here.